checker/jtreg/nullness/issue1275/Main.java

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
You can simply add this jtreg header on top of Crash.java and not add this class.